.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: []
```

Comment:
Seems like something went wrong with some of the PRs?

Reply:
I've made changes to the script so the next wave of PRs won't feature this bug. 🤞